go-selinux/selinux.go

```diff
@@ -213,6 +213,11 @@ func ReserveLabel(label string) {
 	reserveLabel(label)
 }
 
+// IsMLSEnabled checks if MLS is enabled.
+func IsMLSEnabled() bool {
+	return isMLSEnabled()
```

Comment:
I think we're missing a "stub" implementation for `isMLSEnabled()` in `selinux_unsupported.go`; 

<img width="465" alt="Screenshot 2022-10-19 at 13 51 48" src="https://user-images.githubusercontent.com/1804568/196683248-bd5caf51-69c9-4c49-bc0b-8a26b2dd175f.png">


(This may also explain the other Linter issues?)

Reply:
right